src/components/app/details/cicdHistory/Sidebar.tsx

```diff
@@ -115,13 +140,15 @@ const HistorySummaryCard = React.memo(
         const { pathname } = useLocation()
         const currentTab = pathname.split('/').pop()
         const { triggerId, envId, ...rest } = useParams<{ triggerId: string, envId: string }>()
+        const groupCDType: boolean = type === HistoryComponentType.CD || type === HistoryComponentType.GROUP_CD
```

Comment:
The name should be more generic. Currently it seems as it is being true only in case of group_cd